action/bucket.go

```diff
@@ -8,15 +8,37 @@ import (
 	"github.com/yunify/qsctl/v2/contexts"
 )
 
+// BucketHandler is all params for Bucket func
+type BucketHandler struct {
+	*FlagHandler
+	// Remote is the remote qs path
+	Remote string `json:"remote"`
+}
+
+// WithZone rewrite the WithZone method
+func (sh *BucketHandler) WithZone(z string) *BucketHandler {
```

Comment:
What `sh` means here? `bh` looks better for me.

Reply:
That's a typo.